bin/build-sprite-string.js

```diff
@@ -0,0 +1,31 @@
+import defaultAttrs from '../src/default-attrs.json';
+
+/**
+ * Renders a SVG symbol tag
+ * @param {string} name The name of the icon
+ * @param {string} contents The contents of the icon
+ * @returns {string} the rendered SVG symbol
+ */
+function toSvgSymbol(name, contents) {
+  return `    <symbol id="${name}" viewBox="${defaultAttrs.viewBox}">
+      ${contents}
+    </symbol>\n`;
+}
+
+const svgStartTag = `<svg xmlns="${defaultAttrs.xmlns}">
+  <defs>\n`;
+
+const svgEndTag = ' </defs>\n</svg>';
+
+/**
+ * Renders the inner sprites as SVG Symbols
+ * @param {object} icons the icons object
+ * @returns {string} the rendered string with SVG symbols
+ */
+export default function buildSpriteString(icons) {
```

Comment:
I generally like to define the main function (i.e. `buildSpriteString`) at the top of the file and the helper functions (i.e. `toSvgSymbol`) after the main function. I find that it makes files a lot more readable. Using that style this page would be structured something like this:
```
imports...
constants...

function buildSpriteString (...) { ... }
function toSvgSymbol(...) { ...}

export default buildSpriteString
```
Here's a good example of this: https://github.com/feathericons/feather/blob/master/bin/build-icons-object.js